### DISCUSSION TITLE
card-mod, pesce d'aprile

### DISCUSSION THREAD

**Discussion by 50494554524F** [indomus/forum#1608] [HUB: Home Assistant]
se state usando card-mod o qualcosa che usa card-mod le schede di ha appariranno tutte storte e qualcosa incasinato... nessuno problema, pesce d'aprile... poi passa, fortuna che son andato a cercare prima di reinstallare tutto

**Comment by snakuzzo** [comment]
Ah, quindi fammi capire... se non avessi notato questo pesce d'aprile avresti reinstallato tutto ? :D

**Reply by 50494554524F** [reply]
certo, si era aggiornato il supervisor l'altro ieri... ormai faccio prima a reinstallare che risolvere

**Reply by snakuzzo** [reply]
approccio COMPLETAMENTE sbagliato
